core/src/test/java/nl/nn/adapterframework/jdbc/dbms/DbmsSupportTest.java

```diff
@@ -27,115 +27,160 @@
-		assertTrue("Should have found existing column in schema ["+schema+"]", dbmsSupport.isColumnPresent(connection, schema, TEST_TABLE.toLowerCase(), "TINT"));
-		assertTrue("Should have found existing column in schema ["+schema+"]", dbmsSupport.isColumnPresent(connection, schema, TEST_TABLE.toUpperCase(), "TINT"));
-	}
-
-	@Test
-	public void testHasIndexOnColumn() throws JdbcException {
-		String schema = dbmsSupport.getSchema(connection);
-		assertTrue("Should have been index on primary key column", dbmsSupport.hasIndexOnColumn(connection, schema, TEST_TABLE, "TKEY"));
-		assertTrue("Should have been index on primary key column", dbmsSupport.hasIndexOnColumn(connection, schema, TEST_TABLE, "tkey"));
-		assertTrue("Should have been index on column", dbmsSupport.hasIndexOnColumn(connection, schema, TEST_TABLE, "tINT")); // also check first column of multi column index
-		assertFalse("Should not have been index on column", dbmsSupport.hasIndexOnColumn(connection, schema, TEST_TABLE, "TBOOLEAN"));
-		assertFalse("Should not have been index on column", dbmsSupport.hasIndexOnColumn(connection, schema, TEST_TABLE, "tboolean"));
-		assumeThat(productKey, not(anyOf(equalTo("MariaDB"),equalTo("MySQL")))); // MariaDB and MySQL require exact case for table name parameters
-		assertTrue("Should have been index on primary key column", dbmsSupport.hasIndexOnColumn(connection, schema, TEST_TABLE.toLowerCase(), "TKEY"));
-		assertTrue("Should have been index on primary key column", dbmsSupport.hasIndexOnColumn(connection, schema, TEST_TABLE.toUpperCase(), "TKEY"));
-	}
-
-	@Test
-	public void testHasIndexOnColumns() throws JdbcException {
-		String schema = dbmsSupport.getSchema(connection);
+	@DatabaseTest
+	public void testInsertSelect(DatabaseTestEnvironment databaseTestEnvironment) throws JdbcException, SQLException {
+		JdbcUtil.executeStatement(databaseTestEnvironment.getConnection(), "INSERT INTO " + tableName + " (TKEY, TINT) SELECT 11, 2+2 " + databaseTestEnvironment.getDbmsSupport().getFromForTablelessSelect() + " WHERE 1=1");
+	}
+
+	@DatabaseTest
+	public void testIsTablePresent(DatabaseTestEnvironment databaseTestEnvironment) throws JdbcException, SQLException {
+		assertTrue(databaseTestEnvironment.getDbmsSupport().isTablePresent(databaseTestEnvironment.getConnection(), tableName), "Should have found existing table");
+		assertFalse(databaseTestEnvironment.getDbmsSupport().isTablePresent(databaseTestEnvironment.getConnection(), "XXXX"));
+		assumeThat(dataSourceName, not(anyOf(equalTo("MariaDB"), equalTo("MySQL")))); // MariaDB and MySQL require exact case for table name parameters
+		assertTrue(databaseTestEnvironment.getDbmsSupport().isTablePresent(databaseTestEnvironment.getConnection(), tableName.toLowerCase()), "Should have found existing table");
+		assertTrue(databaseTestEnvironment.getDbmsSupport().isTablePresent(databaseTestEnvironment.getConnection(), tableName.toUpperCase()), "Should have found existing table");
```

Comment:
Ik zie dat we hier heel vaak `databaseTestEnvironment.getDbmsSupport()` doen. Laten we dan een property in de class toevoegen `dbmsSupport = databaseTestEnvironment.getDbmsSupport();` als het kan. Dan kun je overal de korte notatie verder gebruiken.